BitwardenShared/UI/Vault/Vault/AutofillList/VaultAutofillListView.swift

```diff
@@ -107,7 +107,35 @@ private struct VaultAutofillListSearchableView: View {
 
     /// A view for displaying a list of ciphers.
     @ViewBuilder
-    private func cipherListView(_ items: [VaultListItem]) -> some View {
+    private func cipherListView(_ sections: [VaultListSection]) -> some View {
+        if store.state.isAutofillingFido2List {
+            cipherCombinedListView(sections)
+        } else {
+            let items = sections[safeIndex: 0]?.items ?? []
```

Comment:
🎨 I think I'd prefer using `sections.first` over the subscript here.
```suggestion
            let items = sections.first?.items ?? []
```